lib/copy/copy_engine.c

```diff
@@ -112,7 +110,8 @@ mem_copy_submit(void *cb_arg, struct spdk_io_channel *ch, void *dst, void *src,
 {
 	struct spdk_copy_task *copy_req;
 
-	rte_memcpy(dst, src, (size_t)nbytes);
+	memcpy(dst, src, (size_t)nbytes);
```

Comment:
In SPDK itself we don't actually use mempcy in any situation where it is performance sensitive*. This was actually the only large copy of variable size remotely close to the I/O path, and this module is only used for testing. Instead, we use zero-copy strategies throughout.

Now, users of SPDK may need to copy data around and are free to use their own high performance memcpy implementations. Since those calls to mempcy are from user code, they don't need SPDK to create a wrapper - they can just call their version directly.

*There are a couple of places where we do a memcpy, but they're small copies of sizes known at compilation time, so the compiler replaces them with the appropriate instructions.